pkg/nvlib/device/device.go

```diff
@@ -208,6 +209,33 @@ func (d *device) IsMigEnabled() (bool, error) {
 	return (mode == nvml.DEVICE_MIG_ENABLE), nil
 }
 
+// HasGpuFabricInfo() checks if a device has GPU Fabric information available.
```

Comment:
Should this be `IsFabricCapable()` to mirror what we do for MIG.

Reply:
done

Reply:
We all discussed offline that this should now be called `IsFabricAttached()` and the implementation will change to reflect those semantics.